Dockerfile.base

```diff
@@ -113,7 +113,8 @@ RUN python3 -m pip install -U --no-cache-dir pip && \
 		-r https://raw.githubusercontent.com/zephyrproject-rtos/zephyr/main/scripts/requirements.txt \
 		-r https://raw.githubusercontent.com/zephyrproject-rtos/mcuboot/main/scripts/requirements.txt \
 		GitPython imgtool junitparser numpy protobuf PyGithub \
-		pylint sh statistics west && \
+		pylint sh statistics west \
+		nrf-regtool && \
```

Comment:
question: how will be versions managed? ie, if vendor requires a new tool version, will we also require a new Zephyr docker image version?

Reply:
Yes, but that is already true with `west` for example.

Reply:
but west can be updated manually in CI (it would not be a vendor specific command). It also has a well defined stable API and release criteria, not sure if vendor tools will have the same